### DISCUSSION TITLE
Add language Support for Malayalam

### DISCUSSION THREAD

**Discussion by neofactoris3** [meilisearch/product#411] [Feedback & Feature Proposal]
Please add language support for [Malayalam](https://en.wikipedia.org/wiki/Malayalam)

**Comment by curquiza** [comment]
Hello @neofactoris3!
Since this is a feature request I moved it to the product repo.
Also, we are really welcome to every kind of contribution to add the support of Malayalam to Meilisearch. This will be handled in this repo: https://github.com/meilisearch/tokenizer
We've just added the Japanese tokenization thanks to the community and this is something we definitely consider for the other languages :)

**Comment by ManyTheFish** [comment]
Hello @neofactoris3,
I put in a new discussion #487 a description of what we could do to enhance Malayalam language support.

(poke @gmourier)

**Comment by gmourier** [comment]
This product discussion is locked in favor of https://github.com/meilisearch/product/discussions/487. Thanks for your help!
